reana_client/cli/quotas.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client quotas related commands."""
+
+import logging
+import sys
+
+import click
+from reana_client.cli.utils import (
+    add_access_token_options,
+    check_connection,
+    NotRequiredIf,
+)
+
+health_color = {"healthy": "green", "warning": "yellow", "critical": "red"}
+
+
+def usage_percentage(usage, limit):
+    if limit == 0:
+        return ""
+    return "({:.1%})".format(float(usage) / float(limit))
+
+
+@click.group(help="Quota commands")
+def quota_group():
+    """Quota commands."""
+    pass
+
+
+@click.option(
+    "--resource",
+    "resource",
+    cls=NotRequiredIf,
+    not_required_if="resources",
+    help="Specify quota resource. e.g. cpu, disk.",
+)
+@click.option(
+    "--resources",
+    "resources",
+    is_flag=True,
+    cls=NotRequiredIf,
+    not_required_if="resource",
+    help="Print available resources",
+)
+@click.option(
+    "--report",
+    "report",
+    type=click.Choice(["limit", "usage"], case_sensitive=False),
```

Comment:
Probably we should tell to use `--resources` to discover the possible values, as this might get outdated.